### DISCUSSION TITLE
Need to refresh dashboard

### DISCUSSION THREAD

**Discussion by vishwa-ims** [RevealBi/Reveal.Sdk#59] [ASP.NET Core]
Hi,

We are using ASP.NET Core SDK and we need to give button in our UI to refresh the dashboard. Currently we can refresh the dashboard by clicking on refresh button that we do have on dashboard screen of SDK but we need to give this custom button outside dashboard

**Comment by TorreyBetts** [comment]
The RevealView in JS has a [refreshDashboardData method](https://help.revealbi.io/api/javascript/latest/classes/revealview.html#refreshdashboarddata) that you can call. It's the equivalent of clicking the Refresh button within our UI.
